From: A. Maitland Bottoms <bottoms@debian.org>
Subject: revert patch cmake: allow UHD library and utilities to be in
 separate lib directories

Users of the module feature expect ${LIB_SUFFIX} to be in the path
for modules. The commit of Wed, 15 Jun 2016 09:44:54 -0700
broke that convention.

--- a/host/lib/utils/CMakeLists.txt
+++ b/host/lib/utils/CMakeLists.txt
@@ -201,18 +201,11 @@
 file(TO_NATIVE_PATH "${CMAKE_INSTALL_PREFIX}" UHD_PKG_PATH)
 string(REPLACE "\\" "\\\\" UHD_PKG_PATH "${UHD_PKG_PATH}")
 
-string(REGEX MATCH ".*/.*" SLASH "${LIB_SUFFIX}")
-if(SLASH STREQUAL "")
-    set(UHD_LIB_DIR "lib${LIB_SUFFIX}")
-else()
-    set(UHD_LIB_DIR "lib")
-endif(SLASH STREQUAL "")
-
 set_source_files_properties(
     ${CMAKE_CURRENT_SOURCE_DIR}/paths.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/images.cpp
     PROPERTIES COMPILE_DEFINITIONS
-    "UHD_PKG_PATH=\"${UHD_PKG_PATH}\";UHD_LIB_DIR=\"${UHD_LIB_DIR}\""
+    "UHD_PKG_PATH=\"${UHD_PKG_PATH}\";UHD_LIB_DIR=\"lib${LIB_SUFFIX}\""
 )
 
 ########################################################################
